packages/pixeloven-webpack/compiler/src/index.ts

```diff
@@ -24,27 +25,27 @@ const defaultCompilerOptions: Config = {
  * @param options
  */
 function getCompiler(options: Partial<Config> = {}) {
+    const { compilers } = options;
+    delete options.compilers;
```

Comment:
@ductiletoaster meant to ask you if this is the best way to avoid passing the compilers on to `getConfig` via the `...config` spread